includes/wc-update-functions.php

```diff
@@ -2175,3 +2175,69 @@ function wc_update_440_insert_attribute_terms_for_variable_products() {
 function wc_update_440_db_version() {
 	WC_Install::update_db_version( '4.4.0' );
 }
+
+/**
+ * Update DB version to 4.5.0.
+ */
+function wc_update_450_db_version() {
+	WC_Install::update_db_version( '4.5.0' );
+}
+
+/**
+ * Get coupons to update in the DB.
+ */
+function wc_update_450_get_coupons() {
+	global $wpdb;
+
+	$coupons = $wpdb->get_results( "SELECT ID, post_title FROM {$wpdb->posts} WHERE post_type = 'shop_coupon';", ARRAY_A );
```

Comment:
This approach has a race condition incase we take too much time in processing these coupons, someone can update the title to something after we have fetched and stored them, and we will end up resetting it from our stored value in options.